d/.gitignore

```diff
@@ -0,0 +1,4 @@
+# unittest files
+*test*
```

Comment:
Why do we need to ignore unittest files?
Are they auto-generated in D?

Reply:
Executable